libs/net/dal/Helpers/FfmpegHelper.cs

```diff
@@ -1,124 +1,152 @@
+using System.Diagnostics;
+using TNO.Core.Extensions;
+
 namespace TNO.DAL.Helpers;
-public class FfmpegHelper
+
+/// <summary>
+/// FfmpegHelper static class, provides helper methods to run ffmpeg commands.
+/// </summary>
+public static class FfmpegHelper
 {
+    #region methods
     /// <summary>
-    /// Get the process for the specified clip command.
+    /// Run the ffmpeg command and create a clip from the file at the specified 'path'.
     /// </summary>
-    /// <param name="fileName"></param>
-    /// <param name="directory"></param>
-    /// <param name="start"></param>
-    /// <param name="end"></param>
-    /// <param name="clipNbr"></param>
-    /// <param name="prefix"></param>
+    /// <param name="path">Full path to the source file.</param>
+    /// <param name="start">Starting position in file.</param>
+    /// <param name="end">Ending position in file.</param>
+    /// <param name="outputName">The name of the file to create.</param>
     /// <returns>System.Diagnostics.Process</returns>
-    public static System.Diagnostics.Process GetClipProcess(string fileName, string directory, int start, int end, int clipNbr, string prefix)
+    public static async Task<string> CreateClipAsync(string path, int start, int end, string outputName)
     {
-        var cmd = GenerateClipCommand(fileName, directory, start, end, clipNbr, prefix);
-        var process = new System.Diagnostics.Process();
-        process = InitializeProcess(process, cmd, "clip");
+        var (Cmd, Output) = GenerateClipCommand(path, start, end, outputName);
+        var process = CreateProcess(Cmd, "clip");
+        var code = await RunProcessAsync(process);
+        // TODO: propogate exception properly.
+        if (code != 0) throw new Exception("An unexpected error occurred while joining clips");
+        return Output;
+    }
 
-        return process;
+    /// <summary>
+    /// Run the ffmpeg command and create a new clip by joining all clips matching the prefix and source file extension type.
+    /// </summary>
+    /// <param name="path">Full path to the source file.</param>
+    /// <param name="prefix">Filename prefix to search for files to join.</param>
+    /// <returns></returns>
+    public static async Task<string> JoinClipsAsync(string path, string prefix)
```

Comment:
Simplified parameters to only require what is needed to join based on convention.  At some point I would like to create a join that takes an array of files, but for now I kept the original functionality.